test/storybook.test.js

```diff
@@ -49,7 +55,11 @@ describe('percy storybook', () => {
   it('snapshots live urls', async () => {
     await storybook(['http://localhost:9000']);
 
-    expect(logger.stderr).toEqual([]);
+    // if there are stderr logs ensure it is only an acceptable warning
+    expect(logger.stderr).toEqual(logger.stderr.length ? [
+      '[percy] Waiting on a response from Storybook...'
```

Comment:
```suggestion
      '[percy] Waiting on a response from Storybook...'
```

why was this required? are we not waiting enough for storybook?

Reply:
We have added extra await for [checkStorybookVersion](https://github.com/percy/percy-storybook/blob/add-storybookv7-support/test/storybook.test.js#L20), this might be causing delays for storybook to start.
Hence, needed to add extra waiting check.

We already do something similar in [storybook-start.test.js](https://github.com/percy/percy-storybook/blob/master/test/storybook-start.test.js#L25)